frontend/src/apiMocks/handlers.ts

```diff
@@ -16,11 +16,11 @@ import {
 } from "./mockData";
 
 export function getHandlers(
-  defaultMockId: null | (typeof mockIds)[number] = null
+  defaultMockId: null | typeof mockIds[number] = null
```

Comment:
I think you'll want to revert the changes in this file and in `mockData.ts` - they're probably a conflict resolution in the wrong direction, and are what's breaking CI. (The reason is a new version of Prettier, which had a small formatting change.)